### DISCUSSION TITLE
blog example not matches with an actual npx-create-app

### DISCUSSION THREAD

**Discussion by supuram** [vercel/next.js#55193] [Help]
### Summary

Dont understand why you use index.js in the blog example but when a newbie creates a new npx create-next-app they are shown pages.js as the main file. Just make it uniform man. Its so disturbing that the symmetry is lost. You work on the blog example for 2 days and now the folder structure changes completely. COmplete madness and loss of symmetry. Disturbing mental behaviour i think. You should go with symmetry always.

**Comment by Ishaan2053** [comment]
Index.js was used with the old `Pages` router. The blog example you mentioned uses the Pages router.

The `App` Router is a newer iteration of the Pages router as quoted in the [documentation](https://nextjs.org/docs/app).

You can adopt either of these Routers for your application, but the App Router is recommended by Vercel.

If you wish to follow the blog example as is, delete your current project folder and run `npx create-next-app` again and make sure to select **NO** when **_'Use App Router?'_** is asked. 

(Also, if the example doesn't use `src` directory, make sure to deselect it too.)
